pkg/api/validation/validation_test.go

```diff
@@ -175,6 +188,10 @@ func TestValidateVolumes(t *testing.T) {
 			if errs[i].(*errors.ValidationError).Field != v.F {
 				t.Errorf("%s: expected errors to have field %s: %v", k, v.F, errs[i])
 			}
+			detail := errs[i].(*errors.ValidationError).Detail
+			if detail != "" && detail != "exactly 1 volume type is required" && detail != DnsLabelErrorMsg {
```

Comment:
Now it's not, it might have been a bug that got fixed, or sth that merge caused, which is not there anymore.